### PR TITLE
MSPSDS-719: Record allegation

### PR DESCRIPTION
**Pull request by tom-bell-softwire**

### CODE REVIEW

File: web/config/initializers/constants.rb [hunk 1]
```diff
@@ -1,3 +1,6 @@
 Rails.application.config.legislation_constants = YAML.load_file(
   Rails.root.join("config", "legislation_constants.yml")
 )
+Rails.application.config.taxonomy_constants = YAML.load_file(
```
Comment (tom-bell-softwire) [inline web/config/initializers/constants.rb]: If you can think of a better name than `taxonomy_constants` that would be great!

File: web/app/controllers/investigations/creation_flow_controller.rb [hunk 2]
```diff
@@ -11,6 +11,7 @@ class Investigations::CreationFlowController < ApplicationController
 
   # GET /xxx/step
   def show
+    set_page_title
```
Comment (broder) [inline web/app/controllers/investigations/creation_flow_controller.rb]: Could this be a `before_action`?

File: web/config/initializers/constants.rb (hunk 1 above)
Comment (broder) [reply web/config/initializers/constants.rb]: Would splitting them out into `hazard_constants` and `product_constants` be neater?

Comment (tom-bell-softwire) [reply web/config/initializers/constants.rb]: Yes it would! Thanks.

File: web/app/controllers/investigations/creation_flow_controller.rb (hunk 2 above)
Comment (tom-bell-softwire) [reply web/app/controllers/investigations/creation_flow_controller.rb]: Good call.

**Pull request merged**